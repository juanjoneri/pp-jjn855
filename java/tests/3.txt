q w e
1 3 __sum__([0,0],[0,1])
2 4 __sum__([1,0],[1,1])
__avg__([0,0],[1,0]) __avg__([0,1],[1,1]) __sum__([0,2],[1,2])
