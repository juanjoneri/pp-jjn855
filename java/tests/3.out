q w e
1 3 4
2 4 6
1.5 3.5 10
